Examcc/src/exo1/vectoriel.java

```diff
@@ -0,0 +1,18 @@
+package exo1;
+
+public class vectoriel {
+	double[] u;
+	double[] v;
+	
+	public static double [] crossProduct(double[] u, double[] v) throws Exception {
+		int size1 = u.length;
+		int size2 = v.length;
+		if (size1 != size2) throw new Exception("Cannot vectoriel cause list size different");
```

Comment:
Ok, this is one error case.
But to be more general, you just needed to check that both u.length=v.length=3